### ISSUE TITLE
[Research] NOD Heuristic Review 

### ISSUE THREAD

**Issue by Evafloyd** [department-of-veterans-affairs/va.gov-team#49816]
## Description
This is 1 of 22 heuristic reviews that will take place to evaluate our set of benefit-related products and features, the [Epic](https://github.com/department-of-veterans-affairs/va.gov-team/issues/49682) provides a broader view. 

At the end of this ticket, our team will have a baseline score for a single product area. 

## Steps
1. Identify our lenses and reviewers
Pulling from the list of VA personas, select 2-4 to use in the review process. Each reviewer will move through the evaluation through the eyes of their assigned persona. These types of reviews are much more fruitful when scoped based on thoughtful prioritization about the actual usage of the thing you are reviewing.

2. Identify scope of review
Identify a set of likely tasks for each of the selected users. These should be tasks in which they might logically encounter the experience we’re reviewing. Try to focus on no more than 5 tasks per user. 
The area of focus should be specific with an outline of defined user tasks. [See list of product areas for running queue](https://docs.google.com/spreadsheets/d/140JebuXONleTpHA5F6dXmKtDhqw9PILeZnnH7BGlEfc/edit#gid=1059373890)

3. Review and prioritize key heuristic measures
Once we have a manageable scope of users and their tasks, we’ll prioritize the principles that we think are of the highest priority for that user for each task. Initially, this will take place in a Google sheet. Note: not all measures may be applicable to the particular persona being used. 

4. Conduct heuristic review
This is the meat of it, 2-4 reviewers per product using the[ documented heuristics](https://docs.google.com/spreadsheets/d/1L6MLFqAnn7FcK4LU4Nh1PCD04lhSZnu84XqHkamEp7E/edit#gid=0)
5. Combine average scores and highlight areas of focus
This step allows us to remove some level of bias by having multiple people review the issues. Once notes have been compared and discussed, we’ll decide as a group which items are most critical for improvement.

## Measures 
In order to take a measured approach to our analysis, we'll employ a quantitative scale to provide quick visualization of how well an application “works.” By providing a scale for this work, we enhance the usefulness of our review. We use the gradations of degree to highlight the overall area. While there isn’t an exact line of demarcation among the values, we want to show more than mere pass or fail; for example, the application might just barely be acceptable in terms of findability, but it could be best in class in terms of clarity.

- Aligns system and real word
- Valuable
- Findable
- Clear
- Trustworthy
- Controlable
- Learnable
- Prevents Errors
- Frictionless

These areas have been adapted from Abby Coverts [Information Architecture Heuristics](https://abbycovert.com/ia-tools/ia-heuristics/), NNGs, and Service Design Heuristics.

### Scale
As you complete the evaluation of your product or service, change the value of the 'Assessment' field for each measure. This field controls the Actual Score and Score %. Possible values for the measure include:

- Unusable: imperative to fix Unusable (0)
- Major issue: Important to fix with high priority Critical (1)
- Minor issue: should be given low priority Moderate (2)
- Cosmetic issue only Minor (3)
- Meets Criteria No Issues (Default Value) (4)

When we create a report, we concentrate on valuation of the main areas. We review the application in as best a manner as we can, scanning it several times from each of the areas. We take a lens-like focus on each area. 
As part of the analysis, we will also provide key examples for each area. However, we generally don’t provide an exhaustive list of every indication of positive or improvable areas

 

## Related Documentation
- TBD

## Tasks
- [x] Identify lenses and reviewers
- [x] Identify the scope of the review (tasks/scenarios)
- [x] Review and prioritize key heuristic measures
- [x] Confirm access to the product area to review on staging
- [x] Conduct heuristic review
- [x] Combine average scores and highlight areas of focus
- [ ] %Create a task for each asset (prototype, wireframe, mural, etc.)%
- [ ] Add assets to the specific test folder and conversation guide

## Acceptance Criteria
- [ ] Review review with the team (particularly product)
- [ ] Publish findings to final location (TBD)
- [ ] %Add any additional Acceptance Criteria%

**Comment by saderagsdale** [comment]
@Evafloyd @eileen-coforma is this still relevant or can we close it?

**Comment by eileen-coforma** [comment]
I think the work is done, but I don't think we have a shared asset within the Github. May need to dig our findings out and convert it to a markdown, so it can be shared.

**Comment by eileen-coforma** [comment]
Work has been added to GH: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/decision-reviews/Notice-of-Disagreement/Research/NoD-Heuristic-Topline-Summary.md
